### DISCUSSION TITLE
[FEATURE REQUEST] Add logging to the NullSoft installer for installation troubleshooting

### DISCUSSION THREAD

**Discussion by doesitblend** [saltstack/salt#67656] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Add logging to the NullSoft installer for installation troubleshooting. This might be helpful? 
https://nsis.sourceforge.io/Dump_log_to_file

**Describe the solution you'd like**
I'm trying to troubleshooting some installation issues on Windows 2016 using the Salt 3004 AMD64 EXE installer and it's quite difficult to figure out what's going on since there are no logs generated.

**Describe alternatives you've considered**
If you use the MSI, the `msiexec` tool is able to generate a log file with expected style of output including the files copied into place, etc. 

#### Sample Command
```
msiexec /i Salt-Minion-3004-Py3-AMD64.msi MASTER=192.168.40.138 MINION_ID=win2016-3 CUSTOM_CONFIG=C:\temp\minion_setup.conf /log minion_install.log /quiet
```

**Comment by doesitblend** [comment]
I chatted with @twangboy about this a little. This depends on support from the NullSoft Installer.

**Comment by marbx** [comment]
Why don't you use the msi instead of the Nullsoft installer?
